### ISSUE TITLE
export * as ns Syntax

### ISSUE THREAD

**Issue by achille1789** [rome/tools#1243]
## Environment

```
Environment Variables
  Object {TERM: "xterm-256color"}

User Config
  unset

Rome Version
  10.0.4-beta

Node Version
  12.16.1

Platform
  darwin x64 19.6.0

Terminal Features
  Object {
    background: "unknown"
    colorDepth: 8
    columns: 201
    cursor: true
    hyperlinks: true
    isTTY: true
    unicode: true
  }

Client Flags
  Object {
    clientName: "cli"
    cwd: "/Users/me/proj/analytics"
    realCwd: "/Users/me/proj/analytics"
    silent: false
  }

Server Status
  Object {
    projects: Array [
      Object {id: 0}
      Object {id: 1}
    ]
    server: Object {
      heapTotal: 26_324_992
      pid: 48_670
      uptime: 0.173209406
    }
    workers: Array [
      Object {
        astCacheSize: 0
        heapTotal: 26_324_992
        ownedBytes: 0
        ownedFileCount: 0
        pid: 48_670
        uptime: 0.17315459
      }
    ]
  }

```

## Steps To Reproduce

1. I have an index file where I am importing/exporting some utilities as documented in here (TS code is compiled with no issues): https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#export--as-ns-syntax
2. Here is the code: export * as utilities from "./utilities";
3. Then I run rome check
4. And I get this log error:  
```
project-rome/@internal/core/server/dependencies/DependencyNode.ts:314:12 internalError/request

  ✖ mod.resolveImport returned NOT_FOUND for an export utilities in /Users/me/proj/analytics/src/index.ts despite being returned by getExportedNames

   6. <anonymous> (project-rome/@internal/core/server/dependencies/DependencyNode.ts:314:12)
   7. formatItem (project-rome/@internal/diagnostics/descriptions/resolver.ts:68:34)
   8. <anonymous> (project-rome/@internal/diagnostics/helpers.ts:43:10)
   9. Array.map
  10. map (project-rome/@internal/diagnostics/helpers.ts:38:25)
  11. UNKNOWN_EXPORT (project-rome/@internal/diagnostics/descriptions/resolver.ts:63:5)
  12. Object.UNKNOWN_EXPORT (project-rome/@internal/diagnostics/descriptions/index.ts:126:51)
  13. ___R$project$rome$$internal$core$server$dependencies$DependencyNode_ts$default.buildDiagnosticForUnknownExport (project-rome/@internal/core/server/dependencies/DependencyNode.ts:306:38)
  14. ___R$project$rome$$internal$core$server$dependencies$DependencyNode_ts$default.resolveImports (project-rome/@internal/core/server/dependencies/DependencyNode.ts:368:27)
  15. ___R$project$rome$$internal$core$server$dependencies$DependencyGraph_ts$default.validate (project-rome/@internal/core/server/dependencies/DependencyGraph.ts:262:31)
  16. ___R$$priv$project$rome$$internal$core$server$linter$Linter_ts$LintRunner.runGraph (project-rome/@internal/core/server/linter/Linter.ts:344:37)
  17. process.processTicksAndRejections (internal/process/task_queues.js:97:4)
  18. ___R$$priv$project$rome$$internal$core$server$linter$Linter_ts$LintRunner.run (project-rome/@internal/core/server/linter/Linter.ts:438:35)
  19. ___R$$priv$project$rome$$internal$core$server$fs$glob_ts$GlobberWatcher.callback (project-rome/@internal/core/server/linter/Linter.ts:532:18)
  20. ___R$$priv$project$rome$$internal$core$server$fs$glob_ts$GlobberWatcher.flush (project-rome/@internal/core/server/fs/glob.ts:241:3)
  21. ___R$$priv$project$rome$$internal$core$server$fs$glob_ts$GlobberWatcher.init (project-rome/@internal/core/server/fs/glob.ts:296:3)
  22. ___R$project$rome$$internal$core$server$fs$glob_ts$Globber.watch (project-rome/@internal/core/server/fs/glob.ts:165:14)
  23. ___R$project$rome$$internal$core$server$linter$Linter_ts$default.runSingle (project-rome/@internal/core/server/linter/Linter.ts:592:21)
  24. ___R$project$rome$$internal$core$server$linter$Linter_ts$default.throwSingle (project-rome/@internal/core/server/linter/Linter.ts:630:20)
  25. Object.callback (project-rome/@internal/core/server/commands/check.ts:130:3)
  and 5 others...

  ⚠ This diagnostic was derived from an internal Rome error. Potential bug, please report if necessary.

```

## Expected Results

lint is passed with no error.

**Comment by sebmck** [comment]
Thanks for the report! I'm happy we invested in decent error messages since it helps a lot with debugging.

**Comment by jamiebuilds** [comment]
Hi! Thank you for opening this issue

We recently made the decision to [migrate the Rome codebase to Rust](https://rome.tools/blog/2021/09/21/rome-will-be-rewritten-in-rust) and making some significant changes to its architecture. We've also decided not to release the previous version written in TypeScript.

We're going through and closing pull requests and issues that were specific to the TypeScript codebase, including this one. Thank you for taking the time to open it.

Thanks again!
